resources/images/view/view.go

```diff
@@ -50,6 +50,11 @@ func (p *RESTView) RenderError(w rest.ResponseWriter, err error, status int, l *
 	rest.Error(w, err.Error(), status)
 }
 
+func (p *RESTView) RenderInternalError(w rest.ResponseWriter, err error, l *log.Logger) {
+	l.F(log.Ctx{log.LogHttpCode: http.StatusInternalServerError}).Error(err.Error())
```

Comment:
removed in next commit: https://github.com/mendersoftware/deployments/pull/160/commits/d69e1312d6595813516f5fd0679c68cbe8633beb#diff-257cd2d9cfbaa0f72550558a1a99ba25L53